app/assets/images/blacklight/list.svg

```diff
@@ -1 +1,5 @@
+<!--
+  Used unmodified from https://fontawesome.com
+  CC BY 4.0 https://creativecommons.org/licenses/by/4.0/
```

Comment:
My understanding was that this came from @jvine not FontAwesome. I'm not sure we need additional comments

Reply:
My bad. Will fix.

Reply:
Done.

Reply:
👍